tasks/JFrogCollectIssues/task.json

```
@@ -59,6 +59,14 @@
             "visibleRule": "configSource=file",
             "helpMarkDown": "Path to a file, with the details required for the build issues collection. Please create a yaml configuration as described <a href=\"https://www.jfrog.com/confluence/display/JFROG/JFrog+Azure+DevOps+Extension#JFrogAzureDevOpsExtension-CollectingBuildIssues\" target=\"_blank\">here."
         },
+        {
+            "name": "workingDirectory",
+            "type": "filePath",
+            "label": "Working Directory",
+            "defaultValue": "",
```

Wouldn't that work? We do this for the build-name and build-number parameters. It explains better the default value.
```suggestion
            "defaultValue": "$(System.DefaultWorkingDirectory)",
```